glean-core/ios/GleanTests/Metrics/EventMetricTests.swift

```diff
@@ -15,6 +15,27 @@ enum ClickKeys: Int32, ExtraKeys {
     }
 }
 
+struct ClickExtras: EventExtras {
```

Comment:
nit: Can this have the same comment as the Kotlin definition? Even that brief explanation that this would usually be generated by glean_parser was useful to know that this was mainly a test construct.

Reply:
Sure, missed that when porting it over.